users/ritvikkolhe/Task 3/Quiz.html

```diff
@@ -0,0 +1,52 @@
+        <br><br><h2>Welcome to IEEE Quiz</h2><br><br>
+        <h4>Q.1 Which of the following is the best club/chapter in VIT ?</h4>
+        <input type="radio" name="" onclick="add()" value="Correct"> IEEE-VIT<br>
+        <input type="radio" name="" value="Incorrect"> CSI<br>
+        <input type="radio" name="" value="Incorrect"> GDG<br>
+        <input type="radio" name="" value="Incorrect"> ACM<br><br>
```

Comment:
This is not a good way to have options, try to check whether the answer is correct or not in the js file itself. Make this change :)